include/galsim/SBInclinedExponential.h

```diff
@@ -69,9 +69,12 @@ namespace galsim {
         /// @brief Returns the scale radius r0 of the disk profile
         double getScaleRadius() const;
 
-        /// @brief Returns the scale height r0 of the disk profile
+        /// @brief Returns the scale height h0 of the disk profile
         double getScaleHeight() const;
 
+        /// @brief Returns the scale height over scale radius h0/r0 of the disk profile
+        double getScaleHOverR() const;
```

Comment:
Starting with the python-layer method, if we allow the user to supply h/r as an input parameter, we should provide a way for them to easily retrieve it, rather than making them figure out how to get it back from other methods.

The reason for implementing it in the C++ layer is efficiency. Doing it this way means only one function call through to that layer is required, rather than two calls and a division operation. It's only a slight gain though, so it might not be worthwhile.